Proposals/XXXX-scaling-codeless-community.md

```diff
@@ -0,0 +1,71 @@
+# Scaling A Codeless Open Source Swift Community with Bas Broek
+
+* Proposal: [PR#22](https://github.com/SwiftCommunityPodcast/podcast/pull/22)
+
+* Authors: [Garric Nahapetian](https://github.com//garricn)
+
+* Review Manager: [Chris Lattner](https://github.com//lattner)
+
+* Status: Proposal Accepted
+
+* Date of Recording: Saturday January 26 2019 10:00 AM PST
+
+## Introduction
+
+Garric, Chris, and Bas will use Bas's experience as the steward of Swift Weekly Brief as a jumping off point to discuss the challenges and lessons learned with scaling a mostly code-less open source Swift community.
+
+## Motivation
+
+The Swift Community Podcast is an experiment and is in its infancy and one of the biggest challenges is how to effectively scale the project through adequate community engagement. A codeless open source community may present challenges that are distinc from code centric ones and so Bas may have some valuable insight into this given his experience with shepherding the Swift Weekly Brief.
+
+## Proposed contribution
+
+I propose to produce the entire episode. However, contributions are welcome. See #Roles section below for open roles.
+
+## Source compatibility
+
+This episode fits with the overall theme of the podcast in the following ways:
+- highlights another Swift community project
+- features a diverse voice in the community
+- increase collaboration among community projects
+- drive contributions to another community project
+
+## Alternatives considered
+
+Bas was already interviewed in the SwiftCoders podcast but the episode was more about Bas rather than his specific work with the Brief. It was also conducted prior to (or very early on) his taking over the Brief. Therefore, this episode is the good format because we can focus on his new experience and tailor the questions in light of this podcast's existence. 
+
+## Roles
+
+- Lead Host: Garric Nahapetian
+
+- Co-hosts: Chris Lattner, Bas Broek
+
+- Editor: open
```

Comment:
Is it too late to volunteer for the editor role? I'm a little bit confused now because the recording date happened before the PR is merged 😅

Reply:
ping @garricn, I think he's currently working on this.

And yes, the episode has already been recorded. We're still figuring out the whole process 😅

Reply:
> Is it too late to volunteer for the editor role? I'm a little bit confused now because the recording date happened before the PR is merged 😅

@VojtaStavik Thank you for offering! Let's see if we can get you to be an editor on the next one.